packages/accordion/src/util.ts

```diff
@@ -15,4 +15,24 @@ export const getAccordionItemMetadata = (item: HTMLElement): AccordionItemMetada
     const value = item.getAttribute("data-accordion-value") ?? "";
     const isItemExpanded = trigger.getAttribute("aria-expanded") === "true"
     return { accordionTriggerElement: trigger, accordionContentElement: content, accordionItemValue: value, isItemExpanded, defaultOpened };
-}
\ No newline at end of file
+}
```

Comment:
Debounce and throttle aren’t really important at this level, as they could create a slight delay in execution. Here, we need the action to happen immediately. The only issue with the collapse/accordion is the state change on the trigger. Ideally, the trigger should change state at the same time as the content expands or collapses, but right now, the trigger only changes state at the end, which isn’t ideal.